### PR TITLE
Adding _list holdings by account_ endpoint

### PR DESCRIPTION
**Pull request by selfnewlin**
This adds the _list holdings by account_ endpoint to the openapi spec.

### CODE REVIEW

File: openapi/mx_platform_api_beta.yml [hunk 1]
```diff
@@ -2693,6 +2693,60 @@ paths:
...
+        required: true
+        schema:
+          type: string
+      - description: The unique id for the `account`.
+        example: ACT-7c6f361b-e582-15b6-60c0-358f12466b4b
+        in: path
+        name: account_guid
+        required: true
+        schema:
+          type: string
```
Comment (brettmortensen) [inline openapi/mx_platform_api_beta.yml:2739]: One request here, would you mind putting this `account_guid` entry as the first entry in this `parameters:` section?

Unfortunately the `normalize` action doesn't alphabetize the params, but I've been doing my best to keep the params in alphabetical order by their `name:` field.

Other than that this looks great 👍

Comment (selfnewlin) [reply openapi/mx_platform_api_beta.yml:2739]: Sure thing.

Comment (selfnewlin) [reply openapi/mx_platform_api_beta.yml]: Done!

**Comment by brettmortensen** [comment]
:shipit:

**Pull request merged**